test-loop-tests/src/tests/catching_up.rs

```diff
@@ -425,99 +405,43 @@ fn test_all_chunks_accepted_common(
         .genesis(genesis)
         .clients(accounts)
         .epoch_config_store(epoch_config_store)
-        .config_modifier(move |config, _| {
-            let block_prod_time = Duration::milliseconds(block_prod_time as i64);
```

Comment:
We got rid of the manual setting for block time as this goes against the principal of testing on production configs.